Artiom/src/main/java/Primitives.java

```diff
@@ -0,0 +1,16 @@
+//3. Write a java program to compare two strings lexicographically, ignoring case differences, if strings are different concatenate a given string to the end of another string.
+
+public class Primitives {
+
+    public static void main(String[] args) {
+        String lower_string = "spring";
```

Comment:
Could you please rename your variables according to naming convention

Reply:
done